Amls 4890 - Page split current address, previous address, other previous address

slavvco opened the pull request:
As above, more in the ticket. New views, controllers, specs for the page split.

## Related / Dependant PRs?

- acceptance tests need updating

## How Has This Been Tested?

- manually 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

ajchilts commented on test/controllers/responsiblepeople/address/AdditionalAddressUKControllerSpec.scala:
```diff
@@ -0,0 +1,326 @@
+
+          status(result) must be(SEE_OTHER)
+
+//          val captor = ArgumentCaptor.forClass(classOf[DataEvent])
```
If we do not need this can we remove it please?

slavvco replied:
done

slavvco commented on app/controllers/responsiblepeople/address/AdditionalExtraAddressNonUKController.scala:
```diff
@@ -0,0 +1,99 @@
+      _ <- OptionT.liftF(AddressHelper.auditPreviousExtraAddressChange(data.personAddress, rp, edit))
+    } yield result
+
+    block getOrElse NotFound(notFoundView)
```
done

slavvco commented on app/controllers/responsiblepeople/address/AdditionalExtraAddressUKController.scala:
```diff
@@ -0,0 +1,101 @@
+      _ <- OptionT.liftF(AddressHelper.auditPreviousExtraAddressChange(data.personAddress, rp, edit))
+    } yield result
+
+    block getOrElse NotFound(notFoundView)
```
done

slavvco commented:
two commits above: first to comply with accessibility requirements -> slight changes to views to include H1 in the legends; and the second one addressed review comments.
